src/index.rs

```diff
@@ -494,15 +494,15 @@ impl IndexTable {
 
 	pub fn drop_file(self) -> Result<()> {
 		drop(self.map);
```

Comment:
I wonder if it would make sense to simulate munmap failure on drop (surely useless here on drop_file, but in general on db close). I mean error on flush will probably be flushed on mmap drop.

Reply:
Yes, indeed. I have a question about it: would the drop panic or just silently fail?

Reply:
I would think silently fail (no handling of error code in https://github.com/RazrFalcon/memmap2-rs/blob/64cf17da2b22fd6fbe6f86346b18906025487891/src/unix.rs#L284 ).

Reply:
actually it is in the comment :facepalm: https://github.com/RazrFalcon/memmap2-rs/blob/64cf17da2b22fd6fbe6f86346b18906025487891/src/unix.rs#L279